app/design/frontend/rwd/default/template/checkout/cart/item/default.phtml

```diff
@@ -108,7 +108,7 @@ $_deleteUrl = $this->getDeleteUrlCustom(false);
                 <a id="<?php echo $helpLinkId ?>" href="#" class="map-help-link"><?php echo $this->__("What's this?"); ?></a>
 
                 <script type="text/javascript">
-                    Catalog.Map.addHelpLink($('<?php echo $helpLinkId ?>'), "<?php echo $this->__("What&#39;s this?") ?>");
+                    Catalog.Map.addHelpLink($('<?php echo $helpLinkId ?>'), "<?php echo Mage::helper('core')->jsQuoteEscape($this->__("What's this?")); ?>");
```

Comment:
would this change break the translations?

Reply:
@fballiano Both `What&#39;s this?` and `What's this?` have translations, so nothing will break.